client/lib/plans/constants.js

```diff
@@ -118,3 +182,11 @@ export const getPlanObject = planName => {
 
 	return objectPlan;
 };
+
+export const getPlanFeaturesObject = planName => {
+	const planFeatures = filter( featuresList, obj =>
+		obj.plans.indexOf( planName ) !== -1
+	);
+
+	return sortBy( planFeatures, 'order' );
```

Comment:
Hmm, I thought we don't want to duplicate the references (as they are currently made from the opposite direction). But I'll implement this as I think it's a better solution. We also don't need the `order` prop in features because we can just order them in the array.